static/src/javascripts/projects/common/modules/identity/upsell/account-creation/AccountBenefits.js

```diff
@@ -0,0 +1,111 @@
+};
+
+const benefits: Benefit[] = [
+    {
+        title: `Get closer to our journalism`,
+        blurb: `Choose from over 40 exclusive newsletters and updates on the variety of topics that matter to you.`,
+    },
+    {
+        title: `Join the conversation`,
+        blurb: `Get involved – post or recommend comments below an article.`,
+    },
+    {
+        title: `Take your career to the next level `,
+        blurb: `Browse a huge variety of jobs, create your jobseeker profile and receive job alerts for vacancies you want to hear about.`,
+    },
+    {
+        title: `Enjoy our free iOS and Android apps`,
+        blurb: `Save articles for later, tailor your news and get notified about topics that matter to you.`,
+    },
+];
+
+const actionableBenefits: ActionableBenefit[] = [
+    {
+        title: `Get closer to our journalism`,
+        blurb: `Browse over 40 curated newsletters on a variety of topics - from daily news headlines to weekly culture highlights, and diverse dedicated topics like science, documentary films and fashion. You can also sign up to receive updates about Guardian events, holidays and offers.`,
+        cta: [
+            {
+                name: `Find your favourite newsletter`,
+                link: `https://${config.get('page.host')}/email-newsletters`,
+            },
+        ],
+    },
+    {
+        title: `Join the conversation`,
+        blurb: `We believe in having open debate and in giving readers a voice in our journalism. With your Guardian profile, you can recommend comments added by other Guardian readers at the bottom of the articles or contribute your thoughts for other readers to see. Complete your public profile and join the debate.`,
+        cta: [
+            {
+                name: `Complete your profile`,
+                link: `${config.get('page.idUrl')}/public/edit`,
+            },
+        ],
+    },
+    {
+        title: `Download our award-winning app`,
+        blurb: `Available on iOS and Android. Sign in with your Guardian account to use exclusive features such as ‘save for later’ or mobile alerts when your favourite writers publish an article, or your football team scores.`,
+        cta: [
+            {
+                name: `Explore our apps`,
+                link: `https://${config.get(
+                    'page.host'
+                )}/technology/ng-interactive/2018/may/15/the-guardian-app`,
+            },
+        ],
+    },
+    {
+        title: `Take your career to the next level`,
+        blurb: `Your account allows you to set up a Guardian Jobs profile, browse a huge variety of jobs and be the first to hear about vacancies in your preferred industry.`,
+        cta: [
+            {
+                name: `Browse jobs`,
+                link: `https://jobs.theguardian.com/`,
+            },
+        ],
+    },
+];
+
+export const AccountActionableBenefits = () => (
+    <ul className="identity-upsell-account-creation-superbullets">
```

Comment:
these classnames are quite long and probably hard to remember, is this the only way this thing can be described?

Reply:
class names are global across all of gu.com so it's best to have them be properly scoped and as long as possible to avoid potential collisions. The name they follow (except for "superbullets" which is an interim name) is the folder path so I'm happy with this